src/client/datascience/notebook/fontAwesomeLoader.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const getPublicPath = () => {
+    return new URL(scriptUrl.replace(/[^/]+$/, '')).toString();
+};
+const uri = getPublicPath() + 'node_modules/font-awesome/css/font-awesome.min.css';
```

Comment:
Does this need to be
```suggestion
const path = require('path');
const uri = getPublicPath() + path.join('node_modules', 'font-awesome', 'css', 'font-awesome.min.css');
```

to ensure the correct path separator is used?

Reply:
normally yes, but in that context the path module can't be found.

Reply:
I don't think we can use `path.join` this is browser code, also `/` (forward slases) works across all OS (yes even windows). No idea why windows supports `\`

Reply:
Super consistent, even with browser urls, its `/`.